### ISSUE TITLE
Animation for each item just show one time

### ISSUE THREAD

**Issue by trinhvanminh2009** [wasabeef/recyclerview-animators#134]
How to scrolling down and scrolling up (scroll all items in list or recycle view) also have animation for items while scrolling ? This animation just happened with new items never load(animation before)

**Comment by danielawde9** [comment]
layout_list_item.xml layout height should be wrap_content not match parent

**Comment by trinhvanminh2009** [comment]
@danielawde9 I don't think so! If you don't believe about that, you should download the source code and check it. Thank you for support.

**Comment by danielawde9** [comment]
Oh I'm sorry I miss read your issue. [late night coding does that]

**Comment by agueroveraalvaro** [comment]
You must see #14 there are a way for the solution
